graph-manager-docs/source/schema-validation.mdx

```diff
@@ -76,19 +76,23 @@ For details, see [Validating changes to the graph](/federation/#validating-chang
 
 ## Using with continuous integration
 
-Schema validation is especially useful when you add it to your continuous integration pipeline (e.g., Jenkins or CircleCI). By doing so, you can detect potential problems automatically and display validation results directly on pull requests.
+Schema validation is especially useful when you add it to your continuous integration pipeline (such as Jenkins or CircleCI). By doing so, you can obtain validation results and display them directly on your team's pull requests.
 
-The `apollo service:check` command returns a non-zero exit code when it detects a breaking change, and it [integrates with GitHub statuses](#integrating-with-github), making it straightforward to use with most CI tools.
+We recommend defining a separate CI job for each [environment (production, staging, etc.)](#validating-against-multiple-environments) you validate schema changes against. The `apollo service:check` command returns a non-zero exit code when it detects a breaking change, meaning the job will fail when validation fails.
```

Comment:
minor: could we be consistent in using the word `variant` or is that too confusing given the vague terminology?

Reply:
Sure! Will update